src/utils/spamprevent.ts

```diff
@@ -0,0 +1,11 @@
+import { Message } from 'discord.js';
+
+import * as config from '../config.json';
+
+export async function messageIsSpam(msg: Message<boolean>): Promise<boolean | undefined> {
+
+    if (msg.mentions.users.size > config.options.maxMentions)
```

Comment:
this can just return the boolean expression, which makes it a one-liner

if this function is only used in one place can get rid of the function and use the conditional directly

Reply:
The reason this is a function is because it's not meant to be singular. it's why it's naming is called `messageIsSpam`
in the future more stuff can be added, like mass emoji detection. or asking of keys, or repeating messages. that kind of stuff.

Reply:
either way, you can just return the boolean expression. dont need an if here